pkg/solana/chainreader/chain_reader_test.go

```diff
@@ -520,7 +520,7 @@ func (r *chainReaderInterfaceTester) GetChainReader(t *testing.T) types.Contract
 
 	require.NoError(t, svc.Start(context.Background()))
 	t.Cleanup(func() {
-		require.NoError(t, svc.Close())
+		_ = svc.Close()
```

Comment:
This check is important. Why are you removing it?
```suggestion
		require.NoError(t, svc.Close())
```

Reply:
Answered here: https://github.com/smartcontractkit/chainlink-solana/pull/829#discussion_r1742553823